src/runtime/config/configuration-qemu-nvidia-gpu.toml.in

```diff
@@ -72,7 +71,7 @@ valid_hypervisor_paths = @QEMUVALIDHYPERVISORPATHS@
 # may stop the virtual machine from booting.
 # To see the list of default parameters, enable hypervisor debug, create a
 # container and look for 'default-kernel-parameters' log entries.
-kernel_params = "@KERNELPARAMS@"
+kernel_params = "@KERNELPARAMS@ @KERNELPARAMS_NV@"
```

Comment:
Completely optional suggestion, but I wonder whether doing:
```
    KERNELPARAMS_NV =  "agent.hotplug_timeout=20"
    KERNELPARAMS_NV += $(KERNELPARAMS)
```
In the `Makefile` and then just `kernel_params = "@KERNELPARAMS_NV@"` would make things clearer/more consistent?

Reply:
Of course I have done that for the TDX case meh